package.json

```diff
@@ -1,36 +1,53 @@
 {
-  "name": "eslint-plugin",
-  "version": "1.0.0",
+  "name": "@trilon/eslint-plugin",
+  "version": "0.0.1",
   "description": "Official Trilon Eslint Plugin",
   "type": "commonjs",
-  "main": "index.js",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "files": [
+    "lib/**/*",
+    "package.json",
+    "README.md",
+    "LICENSE"
+  ],
   "engines": {
     "node": ">=20.9.0",
     "yarn": ">=4.0.2"
   },
   "scripts": {
-    "tsc": "tsc",
+    "prepublish": "tsc",
```

Comment:
I believe we should also make sure all tests are passing and style is correct, thoughts @tuxmachine ?

Reply:
I agree, but that can be enforced in CI right? This is just ensuring that whenever we publish a package, we build a fresh package, and don't accidentally upload an outdated `libs` folder.

Reply:
As of npm@5, `prepublish` scripts are deprecated. Use `prepare` for build steps and `prepublishOnly` for upload-only.